stubs/bleach/bleach/utils.pyi

```diff
@@ -1,8 +1,8 @@
 from collections import OrderedDict
-from typing import Any, Mapping, Text, overload
+from typing import Mapping, Text, overload
 
 def force_unicode(text: Text) -> Text: ...
 @overload
 def alphabetize_attributes(attrs: None) -> None: ...
 @overload
-def alphabetize_attributes(attrs: Mapping[Any, Text]) -> OrderedDict[Any, Text]: ...
+def alphabetize_attributes(attrs: Mapping[Text | None, Text]) -> OrderedDict[Text | None, Text]: ...
```

Comment:
This is dangerous because Mapping is still invariant in its key type. (Changing that is apparently hard; there's an old issue and PR where we tried to make it covariant.) Keeping it as Any is probably better.

Also, looking at `_attr_key` I think the keys should strictly be `tuple[Optional[str], str]`.